src/utils/types.ts

```diff
@@ -62,3 +62,5 @@ export interface Request extends IncomingMessage {
 }
 
 export type Rewriter = (req: Request) => Match | null
+
+export type Location = 'env' | 'flag' | 'config' | 'not found'
```

Comment:
Location is vague and has lots of meaning, let's be more specific here
```suggestion
export type TokenLocation = 'env' | 'flag' | 'config' | 'not found'
```

Reply:
Updated!